.teamcity/Gradle_Check/configurations/FunctionalTest.kt

```diff
@@ -16,10 +16,11 @@ class FunctionalTest(model: CIBuildModel, testCoverage: TestCoverage, subProject
     } + testCoverage.testType.name + "Test"
     val quickTest = testCoverage.testType == TestType.quick
     applyDefaults(model, this, testTask, notQuick = !quickTest, os = testCoverage.os,
+            extraParameters = "-PtestJavaHome=%${testCoverage.os}.${testCoverage.version}.${testCoverage.vendor}.64bit%",
```

Comment:
❌ Please quote the property, e.g. like we do here: https://github.com/gradle/gradle/blob/240521ec79cf094a15f0ae7c3f99ce2702e1801a/.teamcity/Gradle_Check/configurations/GradleBuildConfigurationDefaults.kt#L14

Reply:
👌 ✅